lib/engine.js

```diff
@@ -382,6 +382,7 @@ class GameEngine {
   }
 
   emitEffects(time, effects) {
+    this.listeners.forEach((listener) => {listener.triggered = false});
```

Comment:
Missing semicolon. Also, unless the function callback is going to return something, I think the body of the function should be on it's own separate line to avoid it being misread like `array.map((item) => {item})`.